.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -51,6 +51,9 @@ pipeline {
 
                     // Verify we have the connectivity to Openshift
                     openshift.loginOpenshift()
+
+                    // Fix for rhel8 nodes
```

Comment:
I sent a PR to update jenkins node
I will remove this once the new node with the correction will be live.

Reply:
updated